### PR TITLE
Dbakshani add constant threshold builders

### PR DESCRIPTION
**Pull request by dbakshani**
I have a couple of TODOs left for some improvements. Creating PR for review and feedback.

### CODE REVIEW

**Comment by dbakshani** [comment]
Will do.

**Comment by dbakshani** [comment]
@williewheeler Refactored classes as discussed. Also addressed TODOs and added more tests.

File: adaptivealerting/detectors/builders/constant_threshold.py [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+        detector = ct.ConstantThresholdDetector(STRATEGY, weak_upper_threshold,
+                                                strong_upper_threshold, weak_lower_threshold,
+                                                strong_lower_threshold)
+        LOGGER.info("Detector created -- %s", detector)
+
+        return detector
+
+
+def calculate_sigma(sample):
```
Comment (williewheeler) [inline adaptivealerting/detectors/builders/constant_threshold.py]: Shouldn't this be a "private" method too? (Similarly for the various calculate methods below.)

Comment (dbakshani) [reply adaptivealerting/detectors/builders/constant_threshold.py]: Yes, you are correct. Fixed, thanks.

**Pull request merged**